src/environments/ctmrg_environments.jl

```diff
@@ -460,11 +460,13 @@ function ChainRulesCore.rrule(::typeof(getproperty), e::CTMRGEnv, name::Symbol)
     result = getproperty(e, name)
     if name === :corners
         function corner_pullback(Δcorners)
+            Δcorners = unthunk(Δcorners)
```

Comment:
```suggestion
        function corner_pullback(Δcorners_)
            Δcorners = unthunk(Δcorners_)
```
It tends to be a good idea to not use the same variable name for things that have a different type, as this sometimes confuses the compiler and breaks type inference.